app/views/providers/open_banking_guidances/show.html.erb

```diff
@@ -24,45 +24,61 @@
-
-    <div class="govuk-!-padding-4"></div>
-
-    <h3 class="govuk-heading-m"><%= t('.list_heading_four') %></h3>
-    <p class="govuk-body"><%= t('.your_client_will_see') %></p>
-    <img src="<%= asset_pack_path 'media/images/open_banking/success.png' %>" alt="<%= t('.image_alt_3') %>">
-
-    <div class="govuk-!-padding-4"></div>
-
-    <h3 class="govuk-heading-m"><%= t('.list_heading_five') %></h3>
-    <p class="govuk-body"><%= t('.you_need_to') %></p>
-    <div class="govuk-!-padding-2"></div>
-
-<!--    TODO add this back in later -->
-    <%#= print_button t('.download_button') %>
-<!--    <div class="govuk-!-padding-2"></div>-->
-
-    <%= form.govuk_collection_radio_buttons :can_client_use_truelayer,
-                                            yes_no_options,
-                                            :value,
-                                            :label,
-                                            legend: {text: t(".can_your_client_share"), size: 'm', tag: 'h2'} %>
-
-    <div class="govuk-!-padding-2"></div>
+    <div class="govuk-!-padding-bottom-4 pagebreak">
+      <h3 class="govuk-heading-m"><%= t('.list_heading_two') %></h3>
+      <p class="govuk-body"><%= t('.your_client_selects') %></p>
+      <img src="<%= asset_pack_path 'media/images/open_banking/select_bank.png' %>" alt="<%= t('.image_alt_1') %>" class="no-print">
+      <img src="<%= asset_pack_path 'media/images/open_banking/select_bank.png' %>" alt="<%= t('.image_alt_1') %>" class="only-print open-banking-guidance-img">
+    </div>
+
+    <div class="govuk-!-padding-bottom-4">
+      <h3 class="govuk-heading-m"><%= t('.list_heading_three') %></h3>
+      <p class="govuk-body"><%= t('.your_client_chooses') %></p>
+      <img src="<%= asset_pack_path 'media/images/open_banking/sharing.png' %>" alt="<%= t('.image_alt_2') %>" class="no-print">
+      <img src="<%= asset_pack_path 'media/images/open_banking/sharing.png' %>" alt="<%= t('.image_alt_2') %>" class="only-print open-banking-guidance-img">
+    </div>
+
+    <div class="govuk-!-padding-bottom-4 pagebreak">
+      <h3 class="govuk-heading-m"><%= t('.list_heading_four') %></h3>
+      <p class="govuk-body"><%= t('.your_client_will_see') %></p>
+      <img src="<%= asset_pack_path 'media/images/open_banking/success.png' %>" alt="<%= t('.image_alt_3') %>" class="no-print">
+      <img src="<%= asset_pack_path 'media/images/open_banking/success.png' %>" alt="<%= t('.image_alt_3') %>" class="only-print open-banking-guidance-img">
+    </div>
+
+    <div class="govuk-!-padding-bottom-4">
+      <h3 class="govuk-heading-m"><%= t('.list_heading_five') %></h3>
+      <p class="govuk-body"><%= t('.you_need_to') %></p>
+    </div>
+
+    <div class="govuk-!-padding-bottom-2">
+      <%= print_button t('.print_guidance_button') %>
+    </div>
+
+    <!--    TODO add this back in later -->
+        <%#= print_button t('.download_button') %>
+    <!--    <div class="govuk-!-padding-2"></div>-->
```

Comment:
not you i know but can we get rid of this? if it just does the same thing (print) then the functionality seems redundant in any event.